arangod/Cluster/SynchronizeShard.h

```diff
@@ -44,7 +41,9 @@ class SynchronizeShard : public ActionBase {
 
   virtual ~SynchronizeShard();
 
-  virtual bool first() override;
+  virtual bool first() override final;
+
+  virtual void setState(ActionState state) override final;
```

Comment:
use of "final" implies that unit tests will never override.  bad assumption.

Reply:
when they do, we will change.